src/main/java/subaraki/badbone/events/PlayerUpdateEvent.java

```diff
@@ -104,7 +105,7 @@ public static void playerUpdate(TickEvent.PlayerTickEvent event) {
     @SubscribeEvent
     public static void playerFall(LivingHurtEvent event) {
         if (event.getEntityLiving() instanceof Player player) {
-            if (player.fallDistance > 3 && event.getSource().equals(DamageSource.FALL) && player.getRandom().nextInt(ConfigData.frequencyKnee) == 0) {
+            if (player.fallDistance > 1 && event.getSource().equals(DamageSource.FALL) && player.getRandom().nextInt(ConfigData.frequencyKnee) == 0  && isInSurvivalMode(player)) {
```

Comment:
Why did you change the fall distance from 3 to 1?

Reply:
Fixed now, was while I was testing to make sure things still worked :)